spec/factories/reservation.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :reservation do
+    association :screening
+    email { 'testemail@gmail.com' }
```

Comment:
is a reservation not associated with a user instead of an email

Reply:
user is optional because of the reservation without registering feature